website/blog/2023-03-14-ferretdb-v-0-9-2.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "FerretDB v.0.9.3 - Improved Aggregation Pipeline Support"
+slug: ferretdb-v-0-9-3-improved-aggregation-pipeline-support
+authors: [alex]
+description: >
+  We are thrilled to announce the release of FerretDB v.0.9.3, which includes exciting new features and improvements, and we can't wait for you to try it out
+image: /img/blog/ferretdb_v0.9.3.jpg
+tags: [release]
+---
+
+![FerretDB v0.9.3](/img/blog/ferretdb_v0.9.3.jpg)
+
+We are thrilled to announce the release of [FerretDB v.0.9.3](https://github.com/FerretDB/FerretDB/releases/tag/v0.9.3), which includes exciting new features and improvements, and we can't wait for you to try it out.
+
+<!--truncate-->
+
+This new release is quite monumental for us at FerretDB](https://www.ferretdb.io/), because it brings us one step closer to the release of FerretDB GA, which promises a truly standard open-source document database with MongoDB compatibility, as well as backend engines for [PostgreSQL](https://www.postgresql.org/), [Tigris](https://www.tigrisdata.com/), and potentially many others.
```

Comment:
```suggestion
This new release is quite monumental for us at [FerretDB](https://www.ferretdb.io/), because it brings us one step closer to the release of FerretDB GA, which promises a truly standard open-source document database with MongoDB compatibility, as well as backend engines for [PostgreSQL](https://www.postgresql.org/), [Tigris](https://www.tigrisdata.com/), and potentially many others.
```